third_party/perfetto/protos/perfetto/common/builtin_clock.proto

```diff
@@ -28,9 +28,9 @@ enum BuiltinClock {
   BUILTIN_CLOCK_BOOTTIME = 6;
   BUILTIN_CLOCK_MAX_ID = 63;
 
-  reserved 7, 8;
+  //reserved 7, 8;
```

Comment:
Would you like to turn these comments to !defined()? Or are we able to see if we can re-enable them?

Reply:
So it seemed to me that the protobuf parser that we have checked in couldn't handle reserved fields, and we'd probably have to update it to be able to enable these—as for disabling them, I'm not sure how to do that in protos other than commenting them out like this. Can I do !defined blocks in protos?

Reply:
That is surprising because `tools/protoc`  reports itself as version 3.0.0, which should have support the `reserved` keyword.

Reply:
I'll compile this locally and paste the error here when I have it, maybe you two will have some ideas for it

Reply:
Here's an example failure:

```
[5/3108] ACTION //third_party/perfetto/protos/perfetto/common:lite_gen(//starboard/linux/x64x11/toolchain:target)
FAILED: gen/third_party/perfetto/protos/perfetto/common/android_energy_consumer_descriptor.pb.h gen/third_party/perfetto/protos/perfetto/common/android_energy_consumer_descriptor.pb.cc gen/third_party/perfetto/protos/perfetto/common/android_log_constants.pb.h gen/third_party/perfetto/protos/perfetto/common/android_log_constants.pb.cc gen/third_party/perfetto/protos/perfetto/common/builtin_clock.pb.h gen/third_party/perfetto/protos/perfetto/common/builtin_clock.pb.cc gen/third_party/perfetto/protos/perfetto/common/commit_data_request.pb.h gen/third_party/perfetto/protos/perfetto/common/commit_data_request.pb.cc gen/third_party/perfetto/protos/perfetto/common/data_source_descriptor.pb.h gen/third_party/perfetto/protos/perfetto/common/data_source_descriptor.pb.cc gen/third_party/perfetto/protos/perfetto/common/descriptor.pb.h gen/third_party/perfetto/protos/perfetto/common/descriptor.pb.cc gen/third_party/perfetto/protos/perfetto/common/ftrace_descriptor.pb.h gen/third_party/perfetto/protos/perfetto/common/ftrace_descriptor.pb.cc gen/third_party/perfetto/protos/perfetto/common/gpu_counter_descriptor.pb.h gen/third_party/perfetto/protos/perfetto/common/gpu_counter_descriptor.pb.cc gen/third_party/perfetto/protos/perfetto/common/interceptor_descriptor.pb.h gen/third_party/perfetto/protos/perfetto/common/interceptor_descriptor.pb.cc gen/third_party/perfetto/protos/perfetto/common/observable_events.pb.h gen/third_party/perfetto/protos/perfetto/common/observable_events.pb.cc gen/third_party/perfetto/protos/perfetto/common/perf_events.pb.h gen/third_party/perfetto/protos/perfetto/common/perf_events.pb.cc gen/third_party/perfetto/protos/perfetto/common/sys_stats_counters.pb.h gen/third_party/perfetto/protos/perfetto/common/sys_stats_counters.pb.cc gen/third_party/perfetto/protos/perfetto/common/trace_stats.pb.h gen/third_party/perfetto/protos/perfetto/common/trace_stats.pb.cc gen/third_party/perfetto/protos/perfetto/common/tracing_service_capabilities.pb.h gen/third_party/perfetto/protos/perfetto/common/tracing_service_capabilities.pb.cc gen/third_party/perfetto/protos/perfetto/common/tracing_service_state.pb.h gen/third_party/perfetto/protos/perfetto/common/tracing_service_state.pb.cc gen/third_party/perfetto/protos/perfetto/common/track_event_descriptor.pb.h gen/third_party/perfetto/protos/perfetto/common/track_event_descriptor.pb.cc 
python ../../third_party/perfetto/gn/standalone/protoc.py ./x64/protoc --proto_path ../../third_party/perfetto/ --cpp_out lite=true:gen/third_party/perfetto/ ../../third_party/perfetto/protos/perfetto/common/android_energy_consumer_descriptor.proto ../../third_party/perfetto/protos/perfetto/common/android_log_constants.proto ../../third_party/perfetto/protos/perfetto/common/builtin_clock.proto ../../third_party/perfetto/protos/perfetto/common/commit_data_request.proto ../../third_party/perfetto/protos/perfetto/common/data_source_descriptor.proto ../../third_party/perfetto/protos/perfetto/common/descriptor.proto ../../third_party/perfetto/protos/perfetto/common/ftrace_descriptor.proto ../../third_party/perfetto/protos/perfetto/common/gpu_counter_descriptor.proto ../../third_party/perfetto/protos/perfetto/common/interceptor_descriptor.proto ../../third_party/perfetto/protos/perfetto/common/observable_events.proto ../../third_party/perfetto/protos/perfetto/common/perf_events.proto ../../third_party/perfetto/protos/perfetto/common/sys_stats_counters.proto ../../third_party/perfetto/protos/perfetto/common/trace_stats.proto ../../third_party/perfetto/protos/perfetto/common/tracing_service_capabilities.proto ../../third_party/perfetto/protos/perfetto/common/tracing_service_state.proto ../../third_party/perfetto/protos/perfetto/common/track_event_descriptor.proto
protos/perfetto/common/builtin_clock.proto:31:12: Missing numeric value for enum constant.
Traceback (most recent call last):
  File "/usr/local/google/home/andrewsavage/cobalt/andrewsavage1_cobalt_github/out/linux-x64x11_devel_mono/../../third_party/perfetto/gn/standalone/protoc.py", line 60, in <module>
    sys.exit(main())
             ^^^^^^
  File "/usr/local/google/home/andrewsavage/cobalt/andrewsavage1_cobalt_github/out/linux-x64x11_devel_mono/../../third_party/perfetto/gn/standalone/protoc.py", line 56, in main
    subprocess.check_call(sys.argv[1:])
  File "/usr/lib/python3.11/subprocess.py", line 413, in check_call
    raise CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command '['./x64/protoc', '--proto_path', '../../third_party/perfetto/', '--cpp_out', 'lite=true:gen/third_party/perfetto/', '../../third_party/perfetto/protos/perfetto/common/android_energy_consumer_descriptor.proto', '../../third_party/perfetto/protos/perfetto/common/android_log_constants.proto', '../../third_party/perfetto/protos/perfetto/common/builtin_clock.proto', '../../third_party/perfetto/protos/perfetto/common/commit_data_request.proto', '../../third_party/perfetto/protos/perfetto/common/data_source_descriptor.proto', '../../third_party/perfetto/protos/perfetto/common/descriptor.proto', '../../third_party/perfetto/protos/perfetto/common/ftrace_descriptor.proto', '../../third_party/perfetto/protos/perfetto/common/gpu_counter_descriptor.proto', '../../third_party/perfetto/protos/perfetto/common/interceptor_descriptor.proto', '../../third_party/perfetto/protos/perfetto/common/observable_events.proto', '../../third_party/perfetto/protos/perfetto/common/perf_events.proto', '../../third_party/perfetto/protos/perfetto/common/sys_stats_counters.proto', '../../third_party/perfetto/protos/perfetto/common/trace_stats.proto', '../../third_party/perfetto/protos/perfetto/common/tracing_service_capabilities.proto', '../../third_party/perfetto/protos/perfetto/common/tracing_service_state.proto', '../../third_party/perfetto/protos/perfetto/common/track_event_descriptor.proto']' returned non-zero exit status 1.
[20/3106] ACTION //cobalt/browser:cached_jinja_templates(//starboard/linux/x64x11/toolchain:target)
```